### ISSUE TITLE
Can we remove the prerequisites for the mythic armor ability? 

### ISSUE THREAD

**Issue by Silfarionblade** [Vek17/WrathMods-TabletopTweaks#148]
Specifically, I feel like needing to spend 3 mythic abilities just to get to heavy armor is too much. Especially when you can take archmage armor for one single mythic ability. 

I know I can just cheat it in myself, but I'd like it to be default. Just my 2 cents

**Comment by Amariithynar** [comment]
As in each of the armor mythics should just require the base proficiency and focus?

**Comment by Silfarionblade** [comment]
Yeah that's correct. For example mythic armor (heavy) will require heavy armor proficiency and armor focus heavy armor. That makes it comparable to archmage imo.

**Comment by Amariithynar** [comment]
Agreed. Never much liked how armor requires you to "tier up". Just because you know how to wear and use heavy armor doesn't necessarily mean you know how to use and wear light armor, for example! However, generalized armor training is definitely possible, as with weapons training (aka Martial Weapons Proficiency).

**Comment by Mira132** [comment]
True, the requirements just make it not worth getting for medium and heavy armor characters because you're sacrificing one or two mythic ability picks just to unlock something that is good, but not 2-3 mythic abilities good.

**Comment by Snarling** [comment]
The ability is significantly stronger for heavy armor than it is for light or med. This set of 3 abilities was adapted from one of the tabletop mythic abilities:

Armor Master (Ex)
You don’t take an armor check penalty or incur an arcane spell failure chance when wearing light armor or using a shield (including a tower shield). In addition, the maximum Dexterity bonus of light armor doesn’t apply to you. You can select this ability up to three times. The second time, it also applies to medium armor. The third time, it also applies to heavy armor.

**Comment by Amariithynar** [comment]
> The ability is significantly stronger for heavy armor than it is for light or med. This set of 3 abilities was adapted from one of the tabletop mythic abilities:
> 
> Armor Master (Ex) You don’t take an armor check penalty or incur an arcane spell failure chance when wearing light armor or using a shield (including a tower shield). In addition, the maximum Dexterity bonus of light armor doesn’t apply to you. You can select this ability up to three times. The second time, it also applies to medium armor. The third time, it also applies to heavy armor.

Yes, that's how the tabletop version works, but there's also far more options for armor, as well as far more options to support armor, that make it actually potentially viable to use, since it's part of the Guardian Mythic Path, which heavily focuses on defense and does so at *every* level, in conjunction with all the other Mythic enhancements AND five Mythic feats, while in-game we get JUST the feats, and only a truncated list of them, along with the paths being exceedingly different and more specialized/thematic as well as only paying out in Mythic Path effects only every few Mythic levels. So, unless you think that having Vek to do a full revamp of the Mythic system in the game from the bottom up to match tabletop is reasonable (it really isn't, at least in the short term), making it so that you have to select your specific armor type you want to learn Armor Master for to make up for the reduced Path Ability slots and secondary Mythic effects is reasonable.

**Comment by Mira132** [comment]
> Yes, that's how the tabletop version works, but there's also far more options for armor, as well as far more options to support armor, that make it actually potentially viable to use, since it's part of the Guardian Mythic Path, which heavily focuses on defense and does so at _every_ level, in conjunction with all the other Mythic enhancements AND five Mythic feats, while in-game we get JUST the feats, and only a truncated list of them, along with the paths being exceedingly different and more specialized/thematic as well as only paying out in Mythic Path effects only every few Mythic levels. So, unless you think that having Vek to do a full revamp of the Mythic system in the game from the bottom up to match tabletop is reasonable (it really isn't, at least in the short term), making it so that you have to select your specific armor type you want to learn Armor Master for to make up for the reduced Path Ability slots and secondary Mythic effects is reasonable.

That's a good point. Guardian mythic path gets 10 abilities as well as the 5 mythic feats all mythic paths get if I understand correctly, so for us with just 5 mythic abilities it should be about 1.5 mythic abilities to get armor master with all armor types. Obviously, we can't get 0.5 mythic ability, so they should either be combined into one or find some way to split them into 2 with a little bonus on the second to make it worth it.
